api_tests/src/actors/actor.ts

```diff
@@ -369,6 +369,30 @@ export class Actor {
         await this.swap.doLedgerAction(response.data);
     }
 
+    public async init_does_not_become_available() {
```

Comment:
TypeScript uses camelCase for function names :)

Also, I am not quite sure, whether this is an assertion or not? I don't see an assertion but the name kind of suggests that there should maybe be one?

Reply:
Changed.